GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -499,6 +499,26 @@ public override HashSet<string> UpgradeBlockingProcesses
 
             // Tests show that 250 is the max supported pipe name length
             public override int MaxPipePathLength => 250;
+
+            public override string UpgradeInstallAdvice
+            {
+                get { return "When ready, run `" + this.UpgradeConfirmCommand + "` from an elevated command prompt."; }
+            }
+
+            public override string UpgradeConfirmCommand
+            {
+                get { return "`gvfs upgrade --confirm`"; }

```

Comment:
If you moved `gvfs upgrade --confirm` into a string constant in `WindowsPlatform.Shared.cs ` you could use it in `GetUpgradeReminderNotificationImplementation` as well:

        public static string GetUpgradeReminderNotificationImplementation()
        {
            return "A new version of GVFS is available. Run `gvfs upgrade --confirm` from an elevated command prompt to upgrade.";
        }

Reply:
@jeschu1 I unresolved this conversation.  It looks like this code can be updated to return `UpgradeConfirmMessage` rather than duplicating "'gvfs upgrade --confirm'"